src/core/util/util.js

```diff
@@ -526,6 +526,34 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
       }
     },
 
+    /**
+     * Get an element direct children matching a given tag name
+     *
+     * @param element Element to start walking the DOM from
+     * @param tagName Tag name to match
+     */
+    getChildren: function getChildren(element, tagName) {
```

Comment:
Doesn't look like `getChildren` is being used anywhere. I agree that it may be useful, but it's not worth having it if it's not used.